jupyter_events/logger.py

```diff
@@ -214,44 +140,24 @@ def record_event(self, schema_name, version, event, timestamp_override=None):
         dict
             The recorded event data
         """
-        if not (self.handlers and schema_name in self.allowed_schemas):
+        if not self.handlers or (id, version) not in self.schemas:
```

Comment:
I can see no handlers being defined as fairly common, but I wonder if a missing `(id, version)` warrants some kind of log statement (warning?) since it means events are getting posted without a registered schema (and version match).

Since this could be heavily called in various circumstances, I suppose we'd need to be wary of how much logging it will produce, but I think some level of logging would be helpful in troubleshooting as to why events aren't being seen.

Reply:
This is a good point. It seems like we should log the first time this fails... I'll dive into this more.